apps/wallet-mobile/src/features/Exchange/common/useProviders.tsx

```diff
@@ -0,0 +1,22 @@
+import {Providers} from '@yoroi/exchange'
+import {Exchange} from '@yoroi/types'
+import * as React from 'react'
+
+import {useExchange} from './ExchangeProvider'
+
+export const useProviders = () => {
+  const {orderType} = useExchange()
+
+  const providers = React.useMemo(
+    () => Object.entries(Providers).filter(([, features]) => (orderType === 'buy' ? !!features.buy : !!features.sell)),
+    [orderType],
+  )
+
+  return providers as Entries<Exchange.Providers>
+}
+
+type Entries<T> = Array<
```

Comment:
isn't TS inferring it? or is inferring as any ? 😮‍💨

Reply:
weird error. Not in this component, in the Provider list (flash list)

Not sure if i am doing something wrong